src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: Roboto, sans-serif;
+}
+
+.header__box {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__items {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.header__item {
+  font-size: 12px;
+  line-height: 1;
+  margin: 24px 0;
+}
+
+.header__item + .header__item {
```

Comment:
do you need to increase specificity here?

Reply:
> do you need to increase specificity here?

I don't need him here, you're right

Reply:
> do you need to increase specificity here?

No, but using this method, I indented to the left.Because of this way, the first element ignores the indentation